### PR TITLE
add shedyamls for all devteam's tool collections

### PR DESCRIPTION
**Pull request by martenson**
these seemed to be in pretty good shape but :bird: :eyes: are welcomed as always

### CODE REVIEW

File: tool_collections/cufflinks/cuffmerge/.shed.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/cuffmerge
-owner: devteam
+categories:
+- Transcriptomics
+description: Galaxy wrappers for the Cuffmerge tool.
+homepage_url: http://cole-trapnell-lab.github.io/cufflinks/
 name: cuffmerge
```
Comment (bgruening) [inline tool_collections/cufflinks/cuffmerge/.shed.yml]: long_description: |
  Cuffmerge merges together several Cufflinks assemblies and automatically filters a number of transfrags that are   
  probably artfifacts. If you have a reference GTF file available, you can provide it to the script in order to gracefully merge 
  novel isoforms and known isoforms and maximize overall assembly quality.

File: tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_to_fasta
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (bgruening) [inline tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml]: `Convert Formats`

File: tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+description: FASTQ to FASTA converter
+long_description: |
+  This tool converts data from Solexa format to FASTA format.
```
Comment (bgruening) [inline tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml]: This tool is not only for solexa data. -> from FASTQ format to FASTA

File: tool_collections/fastx_toolkit/fastx_barcode_splitter/.shed.yml [hunk 4]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_barcode_splitter
-owner: devteam
+categories:
+- Fasta Manipulation
+description: Barcode Splitter
+long_description: |
+  This tool splits a Solexa library (FASTQ file) or a regular FASTA
```
Comment (bgruening) [inline tool_collections/fastx_toolkit/fastx_barcode_splitter/.shed.yml]: same here, allowed filetypes are: fasta,fastqsanger,fastqsolexa,fastqillumina

File: tool_collections/fastx_toolkit/fastx_quality_statistics/.shed.yml [hunk 5]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_quality_statistics
-owner: devteam
+categories:
+- Fasta Manipulation
+- Statistics
+description: Compute quality statistics
+long_description: |
+  Creates quality statistics report for the given Solexa/FASTQ library.
```
Comment (bgruening) [inline tool_collections/fastx_toolkit/fastx_quality_statistics/.shed.yml]: Removing Solexa here ...

File: tool_collections/galaxy_sequence_utils/fastq_filter/.shed.yml [hunk 6]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_filter
-owner: devteam
+homepage_url: https://github.com/galaxyproject/sequence_utils
```
Comment (bgruening) [inline tool_collections/galaxy_sequence_utils/fastq_filter/.shed.yml]: Nice!

File: tool_collections/gatk/variant_combine/.shed.yml [hunk 7]
```diff
@@ -1,4 +1,14 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/variant_combine
-owner: devteam
+homepage_url: https://www.broadinstitute.org/gatk/
+categories:
+- Sequence Analysis
```
Comment (bgruening) [inline tool_collections/gatk/variant_combine/.shed.yml]: All GATK tools should have `Variant Analysis` associated and not Sequence Analysis.

File: tool_collections/samtools/bam_to_sam/.shed.yml [hunk 8]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bam_to_sam
-owner: devteam
+categories:
+- SAM
```
Comment (bgruening) [inline tool_collections/samtools/bam_to_sam/.shed.yml]: Convert Formats

File: tool_collections/samtools/sam_to_bam/.shed.yml [hunk 9]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/sam_to_bam
-owner: devteam
+categories:
+- SAM
```
Comment (bgruening) [inline tool_collections/samtools/sam_to_bam/.shed.yml]: Convert Formats

File: tool_collections/samtools/samtools_phase/.shed.yml [hunk 10]
```diff
@@ -1,4 +1,8 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_phase
-owner: devteam
+categories:
+- SAM
```
Comment (bgruening) [inline tool_collections/samtools/samtools_phase/.shed.yml]: Variant Analysis

File: tool_collections/vcflib/vcf2tsv/.shed.yml [hunk 11]
```diff
@@ -1,3 +1,14 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/vcf2tsv
-owner: devteam
+categories:
+- Sequence Analysis
+- Variant Analysis
```
Comment (bgruening) [inline tool_collections/vcflib/vcf2tsv/.shed.yml]: Convert Formats

**Comment by martenson** [comment]
@bgruening All suggestions added even though GitHub does not know. Thanks.

File: tool_collections/bamtools/bamtools/.shed.yml [hunk 12]
```diff
@@ -1,3 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bamtools
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools/.shed.yml]: SAM

File: tool_collections/bamtools/bamtools_filter/.shed.yml [hunk 13]
```diff
@@ -1,3 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bamtools_filter
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools_filter/.shed.yml]: SAM

File: tool_collections/bamtools/bamtools_filter/.shed.yml [hunk 14]
```diff
@@ -1,3 +1,11 @@
...
+description: A tool for filtering on bamfiles
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools_filter/.shed.yml]: s/bamfiles/BAM files/

File: tool_collections/bamtools/bamtools_split/.shed.yml [hunk 15]
```diff
@@ -1,3 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bamtools_split
-owner: devteam
+categories:
+- Sequence Analysis
+description: Tool for splitting bamfiles
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools_split/.shed.yml]: s/bamfiles/BAM files/

File: tool_collections/bamtools/bamtools_split/.shed.yml [hunk 16]
```diff
@@ -1,3 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bamtools_split
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools_split/.shed.yml]: SAM

**Comment by martenson** [comment]
@nsoranzo done

**Comment by bgruening** [comment]
Awesome thanks - nice work!

**Comment by martenson** [comment]
@bgruening thanks for reviews and merge

File: tool_collections/fastx_toolkit/fastq_quality_boxplot/.shed.yml [hunk 17]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_boxplot
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastq_quality_boxplot/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastq_quality_converter/.shed.yml [hunk 18]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_converter
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastq_quality_converter/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastq_quality_filter/.shed.yml [hunk 19]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_quality_filter
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastq_quality_filter/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml (hunk 2 above)
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastq_to_fasta/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastx_artifacts_filter/.shed.yml [hunk 20]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_artifacts_filter
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_artifacts_filter/.shed.yml]: Add "Fastq Manipulation"

File: tool_collections/fastx_toolkit/fastx_barcode_splitter/.shed.yml [hunk 21]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_barcode_splitter
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_barcode_splitter/.shed.yml]: Add "Fastq Manipulation"

File: tool_collections/fastx_toolkit/fastx_clipper/.shed.yml [hunk 22]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_clipper
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_clipper/.shed.yml]: Add "Fastq Manipulation"

**Comment by martenson** [comment]
@nsoranzo I will fix those with the lat batch (suites)

File: tool_collections/fastx_toolkit/fastx_nucleotides_distribution/.shed.yml [hunk 23]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_nucleotides_distribution
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_nucleotides_distribution/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastx_quality_statistics/.shed.yml [hunk 24]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_quality_statistics
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_quality_statistics/.shed.yml]: s/Fasta/Fastq/

File: tool_collections/fastx_toolkit/fastx_renamer/.shed.yml [hunk 25]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_renamer
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_renamer/.shed.yml]: Add "Fastq Manipulation"

File: tool_collections/fastx_toolkit/fastx_reverse_complement/.shed.yml [hunk 26]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_reverse_complement
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_reverse_complement/.shed.yml]: Add "Fastq Manipulation"

File: tool_collections/fastx_toolkit/fastx_trimmer/.shed.yml [hunk 27]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastx_trimmer
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tool_collections/fastx_toolkit/fastx_trimmer/.shed.yml]: Add "Fastq Manipulation"

File: tool_collections/galaxy_sequence_utils/fastq_combiner/.shed.yml [hunk 28]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastq_combiner
-owner: devteam
+categories:
+- Fastq Manipulation
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_combiner/.shed.yml]: Add "Fasta Manipulation"

File: tool_collections/hgv/snpfreq/.shed.yml [hunk 29]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/snpfreq
-owner: devteam
+categories:
+- Variant Analysis
```
Comment (nsoranzo) [inline tool_collections/hgv/snpfreq/.shed.yml]: Add "Statistics"

File: tool_collections/kraken/kraken/.shed.yml [hunk 30]
```diff
@@ -1,14 +1,16 @@
-owner: devteam
-name: kraken
-description: Installs the kraken tool wrapper for taxonomic designation.
 categories:
-  - Metagenomics
-long_description: 'Kraken is a system for assigning taxonomic labels to short DNA
+- Metagenomics
+description: Installs the kraken tool wrapper for taxonomic designation.
```
Comment (nsoranzo) [inline tool_collections/kraken/kraken/.shed.yml]: "Installs" seems more appropriate for the `package_`.

File: tool_collections/kraken/kraken_filter/.shed.yml [hunk 31]
```diff
@@ -1,14 +1,16 @@
-owner: devteam
-name: kraken_filter
-description: Contains a tool wrapper for kraken's confidence filter.
 categories:
-  - Metagenomics
-long_description: 'Kraken is a system for assigning taxonomic labels to short DNA
+- Metagenomics
+description: Contains a tool wrapper for kraken's confidence filter.
```
Comment (nsoranzo) [inline tool_collections/kraken/kraken_filter/.shed.yml]: Remove "Contains a"

File: tool_collections/kraken/kraken_report/.shed.yml [hunk 32]
```diff
@@ -1,14 +1,16 @@
-owner: devteam
-name: kraken_report
-description: Contains galaxy tool wrapper for kraken-report.
 categories:
-  - Metagenomics
-long_description: 'Kraken is a system for assigning taxonomic labels to short DNA
+- Metagenomics
+description: Contains galaxy tool wrapper for kraken-report.
```
Comment (nsoranzo) [inline tool_collections/kraken/kraken_report/.shed.yml]: Remove "Contains"

File: tool_collections/kraken/kraken_translate/.shed.yml [hunk 33]
```diff
@@ -1,14 +1,16 @@
-owner: devteam
-name: kraken_translate
-description: Contains a galaxy tool wrapper for the kraken-translate command.
+homepage_url: http://ccb.jhu.edu/software/kraken/
 categories:
-  - Metagenomics
-long_description: 'Kraken is a system for assigning taxonomic labels to short DNA
+- Metagenomics
+description: Contains a galaxy tool wrapper for the kraken-translate command.
```
Comment (nsoranzo) [inline tool_collections/kraken/kraken_translate/.shed.yml]: Remove "Contains a"

File: tool_collections/samtools/samtools_bedcov/.shed.yml [hunk 34]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_bedcov
-owner: devteam
+categories:
+- SAM
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_bedcov/.shed.yml]: Add "Sequence Analysis"

File: tool_collections/samtools/samtools_flagstat/.shed.yml [hunk 35]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_flagstat
-owner: devteam
+categories:
+- SAM
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_flagstat/.shed.yml]: Add "Statistics"

File: tool_collections/samtools/samtools_idxstats/.shed.yml [hunk 36]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_idxstats
-owner: devteam
+categories:
+- SAM
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_idxstats/.shed.yml]: Add "Statistics"

File: tool_collections/samtools/samtools_mpileup/.shed.yml [hunk 37]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_mpileup
-owner: devteam
+categories:
+- SAM
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_mpileup/.shed.yml]: Add "Variant Analysis"

File: tool_collections/samtools/samtools_sort/.shed.yml [hunk 38]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_sort
-owner: devteam
+categories:
+- Fastq Manipulation
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_sort/.shed.yml]: Remove "Fastq Manipulation"

File: tool_collections/samtools/samtools_stats/.shed.yml [hunk 39]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/samtools_stats
-owner: devteam
+categories:
+- SAM
```
Comment (nsoranzo) [inline tool_collections/samtools/samtools_stats/.shed.yml]: Add "Statistics"

File: tool_collections/taxonomy/poisson2test/.shed.yml [hunk 40]
```diff
@@ -1,4 +1,17 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/poisson2test
-owner: devteam
+categories:
+- Statistics
```
Comment (nsoranzo) [inline tool_collections/taxonomy/poisson2test/.shed.yml]: Add "Metagenomics"

File: tool_collections/vcflib/vcf2tsv/.shed.yml [hunk 41]
```diff
@@ -1,3 +1,15 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/vcf2tsv
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline tool_collections/vcflib/vcf2tsv/.shed.yml]: Remove "Sequence Analysis"

Comment (nsoranzo) [inline tool_collections/vcflib/vcf2tsv/.shed.yml]: "Sequence Analysis" can probably be removed in all `vcflib` repositories.

**Comment by nsoranzo** [comment]
@martenson Thanks, that's perfectly ok for me!

File: tool_collections/hgv/snpfreq/.shed.yml (hunk 29 above)
Comment (bgruening) [inline tool_collections/hgv/snpfreq/.shed.yml]: @nsoranzo I'm not sure a user that is searching for statistic tools expects this tool. As well as the bam-stat tools. Imho these should not be under statistics.

Comment (nsoranzo) [inline tool_collections/hgv/snpfreq/.shed.yml]: I'm fine either way.

**Pull request merged**